? Atlas-0.5.0-compute-elevation-from-btg-files-using-a-multimap-hash.patch
? Atlas-0.5.0-compute-elevation-from-btg-files.patch
? Atlas-0.5.0-disable-bucket-intersection-computation.patch
? Atlas-0.5.0-dont-compute-elevation-while-dragging.patch
? src/data/10
? src/data/4
? src/data/6
? src/data/8
? src/data/9
Index: src/data/Palettes/GMT_globe.ap
===================================================================
RCS file: /cvsroot/atlas/Atlas/src/data/Palettes/GMT_globe.ap,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -p -r1.2 -r1.3
--- src/data/Palettes/GMT_globe.ap	1 Feb 2012 04:35:07 -0000	1.2
+++ src/data/Palettes/GMT_globe.ap	30 Jun 2015 15:50:56 -0000	1.3
@@ -73,6 +73,13 @@ Material Reservoir		water
 Material IntermittentReservoir	water 
 Material Stream			water
 Material Ocean			water
+Material Littoral               water
+Material Estuary                water
+Material Pond                   water
+Material Canal                  water
+Material Lagoon                 water
+Material Watercourse            water
+Material Saline                 water
 
 Material Urban			city
 Material Town			city
Index: src/data/Palettes/NGDC.ap
===================================================================
RCS file: /cvsroot/atlas/Atlas/src/data/Palettes/NGDC.ap,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -p -r1.2 -r1.3
--- src/data/Palettes/NGDC.ap	1 Feb 2012 04:35:07 -0000	1.2
+++ src/data/Palettes/NGDC.ap	30 Jun 2015 15:50:56 -0000	1.3
@@ -57,6 +57,13 @@ Material Reservoir		water
 Material IntermittentReservoir	water 
 Material Stream			water
 Material Ocean			water
+Material Littoral               water
+Material Estuary                water
+Material Pond                   water
+Material Canal                  water
+Material Lagoon                 water
+Material Watercourse            water
+Material Saline                 water
 
 Material Urban			city
 Material Town			city
Index: src/data/Palettes/cd-a.ap
===================================================================
RCS file: /cvsroot/atlas/Atlas/src/data/Palettes/cd-a.ap,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -p -r1.2 -r1.3
--- src/data/Palettes/cd-a.ap	1 Feb 2012 04:35:07 -0000	1.2
+++ src/data/Palettes/cd-a.ap	30 Jun 2015 15:50:56 -0000	1.3
@@ -123,6 +123,13 @@ Material Reservoir		water
 Material IntermittentReservoir	water 
 Material Stream			water
 Material Ocean			water
+Material Littoral               water
+Material Estuary                water
+Material Pond                   water
+Material Canal                  water
+Material Lagoon                 water
+Material Watercourse            water
+Material Saline                 water
 
 Material Urban			city
 Material Town			city
Index: src/data/Palettes/default.ap
===================================================================
RCS file: /cvsroot/atlas/Atlas/src/data/Palettes/default.ap,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -p -r1.2 -r1.3
--- src/data/Palettes/default.ap	1 Feb 2012 04:35:07 -0000	1.2
+++ src/data/Palettes/default.ap	30 Jun 2015 15:50:56 -0000	1.3
@@ -109,6 +109,13 @@ Material Reservoir		water
 Material IntermittentReservoir	water 
 Material Stream			water
 Material Ocean			water
+Material Littoral               water
+Material Estuary                water
+Material Pond                   water
+Material Canal                  water
+Material Lagoon                 water
+Material Watercourse            water
+Material Saline                 water
 
 Material Urban			city
 Material Town			city
Index: src/data/Palettes/wiki-1.03.ap
===================================================================
RCS file: /cvsroot/atlas/Atlas/src/data/Palettes/wiki-1.03.ap,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -p -r1.2 -r1.3
--- src/data/Palettes/wiki-1.03.ap	1 Feb 2012 04:35:07 -0000	1.2
+++ src/data/Palettes/wiki-1.03.ap	30 Jun 2015 15:50:56 -0000	1.3
@@ -67,6 +67,13 @@ Material Reservoir		water
 Material IntermittentReservoir	water 
 Material Stream			water
 Material Ocean			water
+Material Littoral               water
+Material Estuary                water
+Material Pond                   water
+Material Canal                  water
+Material Lagoon                 water
+Material Watercourse            water
+Material Saline                 water
 
 Material Urban			city
 Material Town			city
